cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -150,6 +150,29 @@ def test_embedded_ansible_repository_crud(ansible_repository, wait_for_ansible):
     assert view.description.value == updated_description
 
 
+@pytest.mark.rhel_testing
+@pytest.mark.tier(1)
+def test_embedded_ansible_repository_invalid_url_crud(appliance, wait_for_ansible):
+    """
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    repositories = appliance.collections.ansible_repositories
+    repository = repositories.create(
+        name=fauxfactory.gen_alpha(),
```

Comment:
@sbulage 
As we are using ```appliance.collections.ansible_repositories```  once. It would be better to use it directly while creating repo. like ```appliance.collections.ansible_repositories.create(......)```

Reply:
I have added it because of simplicity.